src/components/DoodleVideo.vue

```diff
@@ -63,31 +96,50 @@ export default {
       return (this.hasFetchedAudio || !this.audioUrl)
         && (this.hasFetchedStrokes || !this.blackboardId)
     },
-    hasVisualAndAudio () { return this.hasFetchedStrokes && this.hasFetchedAudio; }
+    hasVisualAndAudio () { return this.hasFetchedStrokes && this.hasFetchedAudio; },
   },
   watch: {
     mouseHover () { this.$emit("mouse-change", this.mouseHover); },
     hasVisualAndAudio () { this.$emit("visual-audio-ready"); },
-    hasLoadedAvailableResources () { this.$emit("available-resources-ready"); },
+    hasLoadedAvailableResources () { 
+      this.$nextTick(() => { // this waits for audioRecorder to be fully loaded
+        this.$emit("available-resources-ready"); 
+      });
+    },
     // Uncomment below for painless debugging
     // nextFrameIdx () { console.log(`nextFrameIdx = ${this.nextFrameIdx}`)}
   },
   async created () {
     if (!this.thumbnail) {
       this.fetchStrokes();
     } 
+    this.overlay = this.hasBetaOverlay;
   },
   async mounted () {
     this.canvas = document.getElementById(`myCanvas-${this.blackboardId}`);
     this.ctx = this.canvas.getContext("2d");
     await this.setCanvasHeight(); // just for video: blackboard should fill space
     this.$_drawMixin_rescaleCanvas(false);
     window.addEventListener("resize", this.handleResize);
+    if (this.thumbnail) {
+      this.thumbnailImage = document.getElementById(`Thumbnail`);
```

Comment:
```suggestion
      let image = new Image();
      image.src = this.thumbnail  // probably not correct but actual code will be similar
      this.ctx.drawImage(image, 0, 0, this.canvas.scrollWidth, this.canvas.scrollHeight)
```
Then you can clean up the template to only have a canvas.